app/imageswap/imageswap.py

```diff
@@ -274,83 +283,88 @@ def swap_image(container_spec):
 
         app.logger.info('ImageSwap Webhook running in "MAPS" mode')
 
-        swap_maps = build_swap_map(imageswap_maps_file)
+        (exact_maps, swap_maps) = build_swap_map(imageswap_maps_file)
 
         app.logger.debug(f"Swap Maps:\n{swap_maps}")
+        app.logger.debug(f"Exact Maps:\n{exact_maps}")
 
-        # Check if Registry portion includes a ":<port_number>"
-        if ":" in image_registry:
-            image_registry_noport = image_registry.partition(":")[0]
+        if image in exact_maps:
+            app.logger.debug("found exact mapping")
```

Comment:
suggestion (non-blocking): It may be good to print out the map definition here

Reply:
At the end the mapping is logged in info level:
```
    app.logger.info(f"External image definition detected: {image}")
    app.logger.info(f"External image updated to Internal image: {new_image}")
```
So, I think there is no need to debug the exact mapping additionally.